dbadapter/db_adapter.go

```diff
@@ -22,14 +25,18 @@ type DBInterface interface {
 	RestfulAPIPutOneNotUpdate(collName string, filter bson.M, putData map[string]interface{}) (bool, error)
 	RestfulAPIPutMany(collName string, filterArray []primitive.M, putDataArray []map[string]interface{}) error
 	RestfulAPIDeleteOne(collName string, filter bson.M) error
+	RestfulAPIDeleteOneWithContext(collName string, filter bson.M, context context.Context) error
```

Comment:
What about setting the `context` as first parameter to follow GO's common practices?
```suggestion
	RestfulAPIDeleteOneWithContext(context context.Context, collName string, filter bson.M) error
```

Reply:
oh, I did not know about this convention. if I change this here, i would need to make an update also in `utils/mongoapi` otherwise I get an error about not implementing the correct interface. Can we move forward with this PR? and I will make the changes in `utils/mongoapi` to align all the methods and create another PR (way smaller) updating the interface here ?

Reply:
Draft PR for modify order of parameters https://github.com/omec-project/util/pull/109